index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Intro to Pokemon</title>
+</head>
+
+<body>
+    <div class = background>
```

Comment:
Your layout looks good and you're making good use of lists and paragraph tags. Try to incorporate more semantic tags like sections, articles, headers, etc.